scripts/CEE/get-kafka-instance-state/get-kafka-instance-state.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-set -o nounset
```

Comment:
using -nounset is best practice, so we don't favour turning it off.
use the idiom described [here](https://stackoverflow.com/questions/7832080/test-if-a-variable-is-set-in-bash-when-using-set-o-nounset/9824943) instead.

Reply:
I've added `set -o nounset` back, but I think because of the removal/change to the if statement, there are no longer any unset variables anyway - am I misunderstanding that?

Reply:
correct, change looks good. okay to resolve comment.